### DISCUSSION TITLE
Where are transmission-web gui's settings stored?

### DISCUSSION THREAD

**Discussion by helphelpplease** [transmission/transmission#2521] [Q&A]
Hello, previously my queue sort order settings retained in Transmission Web UI.
since about 6 weeks they have stopped. I have not changed anything knowingly in my environment.
I checked configuration file and could not see a setting.

Does anyone know where these settings are stored?

Thankyou For any Assistance
regards

Setting: "Sort Transfers by " in the cog wheel , bottom right
I set mine to reverse order, queue order

screenshot attached

Mac highsierra osx, latest transmission webui

**Comment by helphelpplease** [comment]
sorry, meant to write, cog wheel, bottom left (not bottom right)
https://i.imgur.com/zgRh0N4.png
( I do not know how to attach image as the add link icon is not working)

**Comment by ckerr** [comment]
They're stored in cookies, so if you flush your cookies out the sort order will reset to default.
